UnitTests/Schemas/IFC4X1/beam-varying-profiles/src/beam-varying-profiles.cpp

```diff
@@ -53,17 +51,29 @@ class BeamVaryingProfiles : public VisualTest {
 	}
 
 	virtual std::string TestName() const { return "beam-varying-profiles"; }
-	virtual std::string Schema() const { return "IFC4x1"; }
+	virtual std::string Schema() const { return "IFC4X1"; }
+
+	const boost::filesystem::path filename = dataPath("beam-varying-profiles.ifc");
 
 	std::shared_ptr<oip::EXPRESSModel> express_model = nullptr;
 	buw::ReferenceCounted<oip::IfcImporterT<emt::IFC4X1EntityTypes>> importer = nullptr;
-	buw::ReferenceCounted<oip::IfcGeometryModel> model = buw::makeReferenceCounted<oip::IfcGeometryModel>();
+	buw::ReferenceCounted<oip::IfcModel> model = buw::makeReferenceCounted<oip::IfcModel>();
 };
 
 TEST_F(BeamVaryingProfiles, AllEntitiesAreRead) {
-	EXPECT_THAT(express_model->entities.size(), Eq(63\));
+	EXPECT_THAT(express_model->entities.size(), Eq(63));
+}
+
+TEST_F(BeamVaryingProfiles, IFCHasAnEssentialEntity) {
+	auto result = std::find_if(express_model->entities.begin(), express_model->entities.end(), [](auto &pair) -> bool { return pair.second->classname() == "IFCBEAMSTANDARDCASE"; });
```

Comment:
We focus on geometrical entities, like extruded solids, profile defs and similar. Please check for those as well.

Reply:
Resolved: 95071ab